Resources/Prototypes/Guidebook/species.yml

```
@@ -19,6 +19,7 @@
     - Rodentia
     - Yowie
     - IPC
+    - Oni 
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)